pom.xml

```diff
@@ -117,6 +119,11 @@
          Passed to the JVM running tests. -->
     <jacoco.args></jacoco.args>
     <jacoco.it.args></jacoco.it.args>
+
+    <!-- Intentionally left empty, may be overrided from command line.
+         See README.md and http://errorprone.info/docs/flags for more. -->
+    <java.compiler.errorprone.flag></java.compiler.errorprone.flag>
```

Comment:
Let's make it shorter, sth like errorprone.flag (or flags if multiple are allowed), so that it's easier to use interactively.